### ISSUE TITLE
wing commander: input method not working

### ISSUE THREAD

**Issue by haschimoto** [zer0net/_ppload-zerodos-tickets#3]
Compare : http://playdosgamesonline.com/wing-commander.html
![snap 2016-10-03 at 15 00 57](https://cloud.githubusercontent.com/assets/20817185/19038109/504ec640-897a-11e6-9b63-39b89f1e6fee.png)

**Comment by dnelband** [comment]
im trying to upload different versions, but could be that the dosbox has limitation with this game. in that case i would look for other solutions, like a plugin or a different framework alltogether

**Comment by dnelband** [comment]
it seems more likely that the few files ive tried to upload were all non-playable demo versions, im looking for the real original version and trying again

**Comment by haschimoto** [comment]
Yeah, dosbox clearly works with WC on that dosgames site?

**Comment by dnelband** [comment]
Yes but there they use dosbox on the server and not just js-client side like in our case, i need to test more to see which limitations this has. Perhaps we could develop a python dosbox api for zer0dos
